wp-content/plugins/enhanced-help-menu/plugin.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/*
+Plugin Name: Enhanced help menu
+Description: Shows wordpress help menu on every view and adds "Integreat Hilfe" as first tab that can be via network admin
```

Comment:
Gute Frage :D Ist bei mir in PHP Storm nicht drin.
![enhanced-help-menu](https://cloud.githubusercontent.com/assets/15168372/12419423/54b3c846-beb6-11e5-883b-b24f21ae24e8.jpg)